ci/report_result.yml

```diff
@@ -1,18 +1,52 @@
 ---
-- name: Move callback_plugin result to log dir
-  ansible.builtin.shell:
-    cmd: |
-      cp  ~/test_run_result.out  {{ test_dir }}/test_run_result.log
-     # cp  ~/summary_results.log {{ autoscaling_logs_dir }}/summary_results.log
+- name: Create the output files
+  hosts:
+     - controller
+  vars_files:
+    - vars/common.yml
+  tasks:
+    - name: Create log dir
+      ansible.builtin.file:
+        path: "{{ logs_dir }}/feature-verification-tests/"
+        state: directory
+        mode: "0755"
 
-- name: Grep the number of failed tasks
-  ansible.builtin.shell:
-    cmd: |
-      grep -c "failed" ~/test_run_result.out
-  register: tasks_failed
-  ignore_errors: yes
+- name: Run the log file collection
+  hosts:
+    - controller
+    - compute
+  gather_facts: true
+  vars_files:
+    - vars/common.yml
+  tasks:
+    - name: Collect the custom_logger results
+      delegate_to: controller
+      ansible.builtin.copy:
+        remote_src: true
+        src: "{{ ansible_env.HOME }}/test_run_result.out"
```

Comment:
currently the callback_plugin is saving the files in "~/" not in the {{ ansible_env.HOME }} 

https://github.com/infrawatch/feature-verification-tests/blob/674716b9d70b4e16e5c5a524215e24b68576cc35/callback_plugins/custom_logger.py#L34

Reply:
It's going to be the same in Zuul. But will not always be the same. The callback plugin will use the user's home directory on the ansible controller i.e. where ever the playbooks is called from (hooks in cifmw are run from the controller). It is possible that the you can run the playbook against localhost but with a different user. OR that this would be referenced from a target host.
This SHOULD be the same in our case, however, we could pass in the log dir via ansible.cfg. (might be a good extension to this later). As long as we don't reference the log files from the playbooks running on other hosts (like when grepping the results previously), we'll be fine.

Reply:
Actually, we don't need to copy the files over from other hosts.
We just need to copy them to the logs_dir on the controller, from their other locations.

I think that there might be an issue when we're running multiple playbooks, since the output file will always have the same name, which will get overwritten by any subsequent tests. 
This is not going to be an issue immediately though, since all our jobs only run one test for the moment.

Reply:
For the case where we run multiple tests back-to-back, the custom logger should name the output files according to the play name OR append to an existing file instead of removing it (this could be a config option).